docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: "2.1"
```

Comment:
What is this file needed for? Don't we have `apm-integration-testing` to generate the `docker_compose.yml`?

Reply:
Other agents are providing a docker compose to just test this agent with an Elasticsearch, Kibana and Server for convenience. This is it.

See:
https://github.com/elastic/opbeans-go/blob/master/docker-compose.yml
https://github.com/elastic/opbeans-python/blob/master/docker-compose.yml